YANG/ccamp/layer1-types/ietf-layer1-types.yang

```diff
@@ -252,9 +253,9 @@ module ietf-layer1-types {
       "MEF63: Subscriber Layer 1 Service Attributes";
   }
 
-  identity client-signal {
+  identity cbr-signal {
```

Comment:
it looks I misunderstood the changes. 
I was assuming that we describe the 'client-signal' as the 'CBR' in the 'description' only. Should not change the name here, and afterwards.

Reply:
Fixed with https://github.com/haomianzheng/IETF-ACTN-YANG-Model/pull/106/commits/32f4cc4f7f354b49c73a16e2b1b63b9840bec881